### DISCUSSION TITLE
Can we restrict role "group/offsets/update" to specific consumer group

### DISCUSSION THREAD

**Discussion by dhaddadi** [tchiotludo/akhq#731] [General]
Hello

I wonder if is it possible to restrict the role group/offsets/update with this attribute : consumer-groups-filter-regexp.

We need to give the "modif offset" only for some consumer groups

Thank for advance
Djamel

**Comment by tchiotludo** [comment]
No it's not possible for now I think 🤔
Technically, you can create 2 group, one with the regexp and the right, one other with all the role without  regexp and this role.
But I'm pretty sure that the UI or the backend will miss that.
You can still try and tell me
